packages/backend/.env

```diff
@@ -0,0 +1,7 @@
+# used to point external links at our frontend i.e. in email body.
+VITE_FRONTEND_URL="http://localhost:3000"
+
+# used by emailNotificationService
+VITE_EMAIL_NOTIFIER_HOST=${WG_EMAIL_NOTIFIER_HOST}
+VITE_EMAIL_NOTIFIER_EMAIL=${WG_EMAIL_NOTIFIER_EMAIL}
+VITE_EMAIL_NOTIFIER_PASSWORD=${WG_EMAIL_NOTIFIER_PASSWORD}
```

Comment:
I can help you set up your secrets.  I put some instructions in the readme.md but I'll need to coyp paste our creds to you.